### DISCUSSION TITLE
6.15版本，“系统进程”无数据

### DISCUSSION THREAD

**Discussion by shamohao** [coolsnowwolf/lede#9629] [R68S / 电犀牛专区]
6.15版，系统进程截图如下：

<img width="958" alt="image" src="https://user-images.githubusercontent.com/7984841/174739152-7b3a350f-52e0-4fd0-8361-dad38523612d.png">

**Comment by yangqiceng** [comment]
#9548

**Reply by shamohao** [reply]
OK。可以关闭这个issue

**Comment by stwen3399** [comment]
6.11也一样啊
